pytket/tests/classical_test.py

```diff
@@ -656,7 +652,7 @@ def test_wasmfilehandler_repr_64_2() -> None:
 unsupported function with invalid parameter or result type: 'mixed_up_2' 
 unsupported function with invalid parameter or result type: 'mixed_up_3' 
 unsupported function with invalid parameter or result type: 'unse_internal' 
-"""
+"""  # noqa: W291
```

Comment:
you could add a \n at the end and remove this comment? Or what is the issues here?

Reply:
There are some similar issues around below and above

Reply:
What I mean is:
change the line above to:
```
unsupported function with invalid parameter or result type: 'unse_internal' \n"""
```
and remove this line.

Reply:
If I try that, I get 11 more errors for the other lines above. These would all have to be changed, and the `"""` string replaced by a sequence of concatenated `"` strings. I think this would be less readable.

Reply:
But not sure if that is really a good idea, maybe best to keep it as it is.

Reply:
Yes, keep it as it is! :)